src/block-components/advanced/edit.js

```diff
@@ -52,6 +58,19 @@ export const Edit = () => {
 					title={ __( 'General', i18n ) }
 					id="general"
 				>
+					{ allowCustomAnchor && (
```

Comment:
Ideally, we should only add new things to block components if they are usable by other blocks, but in this case, this option is specific only to the Accordion block. Structurally, this option should not be present here.

That being said, since this option is just an Anchor to the Accordion block, we do not need to create a new option for this because WP already added one under **Advanced Tab > Advanced Panel** as the `anchor` attribute. Let's use that attribute and option instead.